test/export/test_converter.py

```diff
@@ -619,12 +665,11 @@ def forward(self, x: torch.Tensor):
         # Super nested module testing.
         inp = (torch.ones(3),)
         orig_m = SuperNestedM1(3)
-        # TODO: fix trace: state_dict is not equal.
-        ep_list = self._check_equal_ts_ep_converter(orig_m, inp, ["script"])
+        ep_list = self._check_equal_ts_ep_converter(orig_m, inp)
 
         t = inp[0]
         t -= 0.8
-        for ep in ep_list:
+        for ep in ep_list[1:]:
```

Comment:
also slight nit (can do this in a different PR), but maybe instead of returning a list of eps we return a struct so that we could do like, res.ep_script and res.ep_trace instead of indexing